exercises/word-count/instructions.md

```diff
@@ -0,0 +1,73 @@
+# Instructions
+
+Your task is to count how many times each word occurs in a subtitle of a drama.
+
+The dramas that the subtitles are taken from use only ASCII characters.
+
+The characters often speak in casual English, using contractions like _they're_ or _it's_.
+Though these contractions come from two words (e.g. _we are_), the contraction (_we're_) is considered a single word.
+
+Words can be separated by any form of punctuation or whitespace, e.g. "\t", "\n", or " ".
```

Comment:
Maybe also give some examples of punctuation?

Reply:
Good call.

Reply:
I think maybe we can just let them discover through failure that punctuation can repeat.